### PR TITLE
Eslinter update

### PR DESCRIPTION
**Pull request by ButuzGOL**

### CODE REVIEW

File: examples/client/scripts/components/DatepickerExamples.js [hunk 1]
```diff
@@ -16,46 +16,17 @@ export default class DatepickerExamples extends Component {
...
-          <div style={ { width: '20rem' } }>
-            <Datepicker
-              minDate={moment('2016-07-05', 'YYYY-MM-DD').format('YYYY-MM-DD')}
-              maxDate={moment('2016-07-25', 'YYYY-MM-DD').format('YYYY-MM-DD')}
-              selectedDate={ this.state.selectedDate }
-              onSelect={ this.onSelectDate.bind(this) }
-            />
-          </div>
-        </div>
         <h2 className='slds-m-vertical--medium'>Datepicker min date</h2>
```
Comment (stomita) [inline examples/client/scripts/components/DatepickerExamples.js]: Why these DatePicker examples are deleted ?

File: src/scripts/Checkbox.js [hunk 2]
```diff
@@ -38,21 +46,10 @@ export default class Checkbox extends Component {
 }
 
 Checkbox.propTypes = {
-  className: PropTypes.string,
-  label: PropTypes.string,
```
Comment (stomita) [inline src/scripts/Checkbox.js]: The component should accept these properties and it is surely used even if the lint says not used.

File: src/scripts/Checkbox.js [hunk 3]
```diff
@@ -38,21 +46,10 @@ export default class Checkbox extends Component {
...
-    }),
-  ]),
+  error: FormElement.propTypes.error,
   totalCols: PropTypes.number,
   cols: PropTypes.number,
-  name: PropTypes.string,
-  value: PropTypes.any,
   grouped: PropTypes.bool,
-  checked: PropTypes.bool,
-  defaultChecked: PropTypes.bool,
```
Comment (stomita) [inline src/scripts/Checkbox.js]: The component should accept these properties adn it is surely used even if the lint says not used.

**Comment by ButuzGOL** [comment]
@stomita agreed I just missed it (

File: examples/client/scripts/components/DatepickerExamples.js [hunk 4]
```diff
@@ -29,9 +29,9 @@ export default class DatepickerExamples extends Component {
         <div style={ styles }>
           <div style={ { width: '20rem' } }>
             <Datepicker
-              minDate={moment('2016-07-05', 'YYYY-MM-DD').format('YYYY-MM-DD')}
-              maxDate={moment('2016-07-25', 'YYYY-MM-DD').format('YYYY-MM-DD')}
-              selectedDate={ this.state.selectedDate }
+              minDate={ moment('2016-07-05', 'YYYY-MM-DD').format('YYYY-MM-DD') }
+              maxDate={ moment('2016-07-25', 'YYYY-MM-DD').format('YYYY-MM-DD') }
+              selectedDate={ moment('2016-07-10', 'YYYY-MM-DD').format('YYYY-MM-DD') }
```
Comment (stomita) [inline examples/client/scripts/components/DatepickerExamples.js]: The selectedDate changed to a statically defined value, not reflecting state change.

**Comment by ButuzGOL** [comment]
@stomita please merge so provide next pr

**Pull request merged**